app/services/importers/ok_sos/split_files.rb

```diff
@@ -0,0 +1,121 @@
+module Importers
+  module OkSos
+    class SplitFiles < ApplicationService
```

Comment:
this file should be split into two...one for splitting and one for importing.
I'm using tmp files which means keeping the @csvs variable in memory is important and splitting might give you issues
tbh though I think if you use a spawned dyno all files behave as temp files? Idk..